service-runner.js

```diff
@@ -54,6 +56,9 @@ class ServiceRunner {
     }
 }
 
+ServiceRunner.getLogger = (loggerConf) => { return new Logger(loggerConf); };
```

Comment:
I'd prefer method-like syntax within the class

```
static getLogger(loggerConf) { return new Logger(loggerConf); }

Reply:
Oh, sure, I was unfamiliar with that syntax.  Will fix.